detekt-rules-complexity/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/complexity/CyclomaticComplexMethod.kt

```diff
@@ -38,15 +38,17 @@ import org.jetbrains.kotlin.psi.KtWhenExpression
  *  [Reference](https://kotlinlang.org/docs/scope-functions.html)
  */
 @ActiveByDefault(since = "1.0.0")
-class ComplexMethod(config: Config = Config.empty) : Rule(config) {
+class CyclomaticComplexMethod(config: Config = Config.empty) : Rule(config) {
 
     override val issue = Issue(
-        "ComplexMethod",
+        "CyclomaticComplexMethod",
         Severity.Maintainability,
         "Prefer splitting up complex methods into smaller, easier to understand methods.",
```

Comment:
```suggestion
        "Prefer splitting up complex methods into smaller, easier to test methods.",
```

Cyclomatic focus more on testability. This also make it a bit more different from cognitive